mpf/config_spec.yaml

```diff
@@ -697,6 +697,12 @@ counters:
     count_interval: single|int|1
     direction: single|str|up
     starting_count: single|template_int|0
+    control_events: list|subconfig(counter_control_events)|None
+counter_control_events:  # subconfig for mode counters
+    __valid_in__: None
+    action: single|enum(add,subtract,set_value)|
+    event: single|str|
+    value: single|int|None  # int for all
```

Comment:
You could use template_int here. That would allow placeholders which are evaluated during runtime (whem you call .evaluate(additional_vars) on them). Could also be a separate future change.